.travis.yml

```diff
@@ -13,7 +13,7 @@ before_script:
 - sudo mount --make-rshared /
 
 script:
-- shellcheck scripts/*.sh
+- shellcheck scripts/*.sh conf/sample/*.sh
```

Comment:
Can we switch to this?

find . -name "*.sh" -not -path "./vendor/*" | xargs shellcheck

Reply:
Or even this:

` shellcheck $(find . -name "*.sh" -not -path "./vendor/*")`

I'll re-spin.